### PR TITLE
Include sjk-plus for extracting metric on OSS C*

### PR DESCRIPTION
**Pull request by adejanovski**
Fix regexes which didn't allow to extract the node address if there are trailing whitespaces.

### CODE REVIEW

**Comment by adejanovski** [comment]
Tested successfully on a tlp-cluster generated cluster.

**Comment by msmygit** [comment]
@adejanovski, did you also test this against the regular DSE cluster?

**Comment by adejanovski** [comment]
I didn't because I don't have an easy way to spin them up.

File: collect_diag.sh [hunk 1]
```diff
@@ -153,6 +153,10 @@ declare -A servers
 for host in $(cat "$HOST_FILE"); do
     debug "Copying collect_node_diag.sh to $host..."
     scp $SSH_OPTS "${LAUNCH_PATH}collect_node_diag.sh" "${host}:~/"
+    if [ "$TYPE" = "coss" ]; then
+        debug "Copying sjk jar to $host..."
+        scp $SSH_OPTS "${LAUNCH_PATH}/libs/sjk-plus.jar" "${host}:~/"
```
Comment (msmygit) [inline collect_diag.sh:158]: #nit We don't need the `/` character between `${LAUNCH_PATH}` & `lib/sjk-plus.jar` here as we already have it in here, https://github.com/DataStax-Toolkit/diagnostic-collection/blob/master/collect_diag.sh#L153 as `[[ $0 == */* ]] && LAUNCH_PATH=${0%/*}/ || LAUNCH_PATH=./`

File: collect_node_diag.sh [hunk 2]
```diff
@@ -757,6 +758,13 @@ function collect_data {
         fi
     fi
 
+    # Collect metrics from JMX for OSS C* and DDAC
+    if [ -n "$IS_COSS" ] ; then
```
Comment (msmygit) [inline collect_node_diag.sh:762]: Curious why is this specific to OSS and DDAC only. Can't we also leverage this for DSE?

Comment (adejanovski) [reply collect_node_diag.sh:762]: DSE (and Cassandra 4.0) has a nodetool command which allows to invoke sjk while OSS C* < 4.0 doesn't have that feature. That's why we just need this for OSS.
Using it for DSE is possible but metrics extraction is already covered.

I'm not sure about DDAC though 🤔  but using sjk shouldn't hurt there even if the nodetool sjk command is available.

Comment (msmygit) [reply collect_node_diag.sh:762]: DDAC is DSE 5.1 so, I'm thinking this shouldn't be a problem at all for any version of DSE 🤔

**Review by msmygit — APPROVED** [review]

**Pull request merged**